core/src/main/scala/cats/effect/TestConsole.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect
+import cats.Show
+import cats.effect.concurrent.Ref
+import cats.syntax.show._
+
+class TestConsole(out1: Ref[IO, List[String]],
```

Comment:
It should probably use `Chain`

Reply:
Also, I made it a non-case class because the only case-classy feature that was used was the `apply` method

Reply:
Obviously it needs to be parameterized with `F[_]` :)